lego/apps/events/permissions.py

```diff
@@ -9,6 +10,56 @@
 class EventPermissionHandler(PermissionHandler):
 
     perms_without_object = [CREATE, 'administrate']
+    authentication_map = {VIEW: False, LIST: False}
+    force_queryset_filtering = True
+
+    def filter_queryset(self, user, queryset, **kwargs):
+        if not user.is_authenticated or not user.is_abakom_member:
+            return queryset.filter(can_view_groups=None)
+        return queryset
+
+    def has_perm(
```

Comment:
This is duplicating the `permissions/permissions.py` quite extensively, I think its better to do the abakom check explicitly and do super().has_perm to reuse the default logic. 

Maybe something like this is better?
```
can_view_event = True
if obj and isinstance(obj, ObjectPermissionModel) and obj.is_abakom_only:
   can_view_event = user.is_abakom_member
return can_view_event and super().has_perm(*args, **kwargs)
```